services/platform/platform.go

```diff
@@ -564,18 +564,22 @@ func (s *Service) ServerInfo(ctx context.Context, req *platformpb.ServerInfoRequ
 	}
 
 	serverName := ""
+	var connectedToPortal bool
```

Comment:
nit: I'd write
`connectedToPortal := false`

but it just a personal preference. this way we are not relying no default the value (false), and just being more explicit

Reply:
changed, thanks